### ISSUE TITLE
False error message positives

### ISSUE THREAD

**Issue by 6elkir** [iCalcreator/iCalcreator#80]
In vendor/kigkonsult/icalcreator/src/Vcalendar.php:943 error 91 false positives then i trying parse Booking.com calendar. This code get error 91:

        ```
        $result = new \Kigkonsult\Icalcreator\Vcalendar
        (
            array
            (
                \Kigkonsult\Icalcreator\Vcalendar::ALLOWEMPTY => true,
                \Kigkonsult\Icalcreator\Vcalendar::UNIQUE_ID  => static::$_provider
            )
        );

        $result -> parse ( $content );

        return $result;
        ```

\Kigkonsult\Icalcreator\Vcalendar::ALLOWEMPTY did not help solve the problem.

**Comment by iCalcreator** [comment]
Also noted, fix asap !

**Comment by iCalcreator** [comment]
The ALLOWEMPTY config manages properties, not components.

Can't reproduce, will you please send over the calendar file in question?

**Comment by 6elkir** [comment]
Sorry, im parse booking.com & airbnb.ru on production and this calendars already rebuild next dates =\

**Comment by 6elkir** [comment]
I suspect that these are calendars without VEVENT components

**Comment by iCalcreator** [comment]
Well, an empty collection is also a collection ??

Ok, will remove the parse component count test (ie no exception).

**Comment by iCalcreator** [comment]
Fixed in 2.28.2
